docs/guides/index.md

```diff
@@ -260,7 +262,7 @@ created by creating a sphere that wraps the scene:
 
 [next]: ./build-with-components.md
 
-And that is a very basic scene that places static objects in a 3D space using
-HTML. A good VR experience requires rich interaction and dynamic behavior. With
-the help of using and writing A-Frame components, we can [build a more advanced
+That is all it takes to create a very basic scene which places static objects in a 3D space using
+HTML! A good VR experience requires rich interaction and dynamic behavior. With
```

Comment:
not sure if hello world is worthy of hype. i sort of preferred the intention before, "that was a basic scene, now we are moving on to do cooler things" rather than "wasn't that amazing?"